package castor.messages;

message Heartbeat {
 required int64 bytesMoved = 1;
}
